shared/src/Shared-Src/README.MD

```diff
@@ -0,0 +1,8 @@
+﻿This directory and its subdirectories contains source files that are stored in this repo and are used across several projects.
+
+One of the reasons for doing it is to share the implementation across projects for APIs that should
+not be public and therefore cannot be grouped into a separate shared assembly.
+
+Formanaged code, the subdirectories here carry the name of the respective namespaces.

```

Comment:
```suggestion
For managed code, the subdirectories here carry the name of the respective namespaces.
```

Reply:
Addressed in https://github.com/DataDog/dd-trace-dotnet/commit/8b9642541f0ad21873f3bdd9325b6601a10bb8c9